### ISSUE TITLE
Failed to precompile test:test

### ISSUE THREAD

**Issue by onemanstartup** [exercism/dart#230]
I have last version of dart. Last version of exercism and last two-fer code.
```dart
~/Code/exercism/dart/two-fer master* 7s
❯ exercism download --exercise=two-fer --track=dart

Downloaded to
/Users/j/Code/exercism/dart/two-fer

~/Code/exercism/dart/two-fer master*
❯ pub get
Resolving dependencies...
Got dependencies!
Precompiling executables... (3.4s)
Failed to precompile test:test:
Error: Could not resolve the package 'kernel' in 'package:kernel/util/graph.dart'.
../../../../.cache/dart_cache/hosted/pub.dartlang.org/analyzer-0.39.2/lib/src/summary2/default_types_builder.dart:15:8: Error: Not found: 'package:kernel/util/graph.dart'
import 'package:kernel/util/graph.dart' show Graph, computeStrongComponents;
       ^
../../../../.cache/dart_cache/hosted/pub.dartlang.org/analyzer-0.39.2/lib/src/summary2/default_types_builder.dart:256:39: Error: Type 'Graph' not found.
class _TypeParametersGraph implements Graph<int> {
                                      ^^^^^
../../../../.cache/dart_cache/hosted/pub.dartlang.org/analyzer-0.39.2/lib/src/summary2/default_types_builder.dart:256:39: Error: Expected 0 type arguments.
class _TypeParametersGraph implements Graph<int> {
                                      ^
../../../../.cache/dart_cache/hosted/pub.dartlang.org/analyzer-0.39.2/lib/src/summary2/default_types_builder.dart:125:29: Error: The method 'computeStrongComponents' isn't defined for the class 'DefaultTypesBuilder'.
 - 'DefaultTypesBuilder' is from 'package:analyzer/src/summary2/default_types_builder.dart' ('../../../../.cache/dart_cache/hosted/pub.dartlang.org/analyzer-0.39.2/lib/src/summary2/default_types_builder.dart').
Try correcting the name to the name of an existing method, or defining a method named 'computeStrongComponents'.
    var stronglyConnected = computeStrongComponents(graph);
                            ^^^^^^^^^^^^^^^^^^^^^^^

~/Code/exercism/dart/two-fer master*
❯ dart --version
Dart VM version: 2.6.1 (Mon Nov 11 13:12:24 2019 +0100) on "macos_x64"```

**Comment by Stargator** [comment]
Hey @onemanstartup, the `kernel` package was removed and they released 0.39.2+1 of `analyzer`.

Try running `pub upgrade` and see if they pulls down the latest dependencies. Hopefully, that resolves your issue.

**Comment by onemanstartup** [comment]
@Stargator Thank you! It works.
